nav2_dwb_controller/README.md

```diff
@@ -27,10 +27,10 @@ The advantages of DWB were:
 
 For the most part, the DWB codebase is unchanged, other than what was required for ROS2.
 
-The main architectural change is due to the replacement of `MoveBase` by the `FollowPath` action server.
+The main architectural change is due to the replacement of `MoveBase` by the `FollowPath` action server, called the controller_server, which uses this DWB plugin.
```

Comment:
Perhaps splitting hairs, but MoveBase was replaced by the BtNavigator, which interfaces to the various action servers - ComputePathToPose, FollowPath, recoveries, etc. - using action clients.

Reply:
This is in context of the nav2 dwb controller readme, so I think it makes sense that this was pulled out of movebase, but what you said probably makes more sense

Reply:
@mjeronimo I think that entire sentence doesn't provide much value. I'm just deleting it, its in a section discussing DWB changes for ROS2 and none of that should be important for a plugin to understand or care about.